server/middleware/caseloadCheckMiddleware.test.ts

```diff
@@ -1,5 +1,5 @@
 import caseloadCheck from './caseloadCheckMiddleware'
-import { mockNext, mockRequest, mockResponse } from '../routes/testutils/requestTestUtils'
+import { mockNext, mockRequest, mockResponse } from '../routes/__testutils/requestTestUtils'
```

Comment:
moved test utils into a sub directory to make it distinct compared to the other folders that represent different parts of the app